From 02fe68e28e492ae9498ac32664d0831245dc6c07 Mon Sep 17 00:00:00 2001
From: Richard Liu <r66033@freescale.com>
Date: Mon, 12 Oct 2015 20:12:11 +0800
Subject: [PATCH 06/22] MA-5839 Removed the DANDROID_P2P_STUB

We have our private lib and no need the P2P_STUB.

Signed-off-by: Wang Haoran <b50027@freescale.com>

Conflicts:
	hostapd/Android.mk
	wpa_supplicant/Android.mk

Conflicts:
	wpa_supplicant/Android.mk

Change-Id: I59b9a5ebb8e4a52eb255cf2f54fc33ab5f918712
---
 hostapd/Android.mk        | 3 ---
 wpa_supplicant/Android.mk | 4 ----
 2 files changed, 7 deletions(-)

diff --git a/hostapd/Android.mk b/hostapd/Android.mk
index 52d4cfe..3232630 100644
--- a/hostapd/Android.mk
+++ b/hostapd/Android.mk
@@ -30,9 +30,6 @@ L_CFLAGS += -Wno-unused-parameter
 # Set Android extended P2P functionality
 L_CFLAGS += -DANDROID_P2P
 
-ifeq ($(BOARD_HOSTAPD_PRIVATE_LIB),)
-L_CFLAGS += -DANDROID_LIB_STUB
-endif
 
 # Use Android specific directory for control interface sockets
 L_CFLAGS += -DCONFIG_CTRL_IFACE_CLIENT_DIR=\"/data/misc/wifi/sockets\"
diff --git a/wpa_supplicant/Android.mk b/wpa_supplicant/Android.mk
index 560c982..9356b6c 100644
--- a/wpa_supplicant/Android.mk
+++ b/wpa_supplicant/Android.mk
@@ -28,10 +28,6 @@ L_CFLAGS += -Wno-unused-parameter
 # Set Android extended P2P functionality
 L_CFLAGS += -DANDROID_P2P
 
-ifeq ($(BOARD_WPA_SUPPLICANT_PRIVATE_LIB),)
-L_CFLAGS += -DANDROID_LIB_STUB
-endif
-
 # Disable roaming in wpa_supplicant
 ifdef CONFIG_NO_ROAMING
 L_CFLAGS += -DCONFIG_NO_ROAMING
-- 
1.8.0

